src/Entity/Timeline/Measure.php

```diff
@@ -307,4 +298,28 @@ public function profileIds(): array
             return $profile->getId();
         }, $this->profiles->toArray());
     }
+
+    /**
+     * @Algolia\Attribute
+     */
+    public function title(): array
```

Comment:
Can you rename it `titles`?

Reply:
i can on my end yes.
i will wait for approval of the front who needs this algolia attribute under this name though